### DISCUSSION TITLE
How to open an existing document?

### DISCUSSION THREAD

**Discussion by RevealedFrom** [dolanmiu/docx#1265] [Q&A] [answered]
Forgive me if this is already obvious. I looked through the examples, but can't seem to find the means to open an existing docx document. I see only `ImportDotx`. How do I load an existing docx document into an object so that I can modify it? Thanks.

**Comment by dolanmiu** [comment, marked as answer]
not possible im afraid

This library only allows for creation of docx files

**Reply by jordanparker6** [reply]
@dolanmiu do you have any good recommendations for loading?

Looks like it isn't a trivial problem...
